### DISCUSSION TITLE
503 service unavailable

### DISCUSSION THREAD

**Discussion by SantosJaimes** [YOURLS/YOURLS#3572] [Q&A]
Hello,

I have configured config.php and uploaded all files.  Now, I am getting a 503 error:

#############
Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.
#############

I am using a virtual server, and other sites load just fine.

Any suggestions? 

Thanks in advance.

**Comment by ehthxC7x9D** [comment]
I would suggest making sure you have stepped through the most common troubleshooting issues first (https://yourls.org/docs/category/troubleshooting) and then, if you are still having problems please bring your logs and environment info.  This one I would check your HTTP daemon error logs for details about the 503.  Feel free to post them here with sensitive info removed; they will help us, help you.
